frontend/packages/core/src/Assets/colortypes.tsx

```diff
@@ -0,0 +1,113 @@
+export interface NeutralColors {
+  5: string; // PRIMARY BUTTON DISABLED / STROKE TERTIARY
+  10: string;
+  12: string; // SECONDARY BUTTON PRESSED / FOCUSED
+  15: string;
+  20: string;
+  25: string;
+  30: string; // STROKE SECONDARY / TEXT TERTIARY
+  40: string;
+  50: string; // TEXT SECONDARY
+  60: string;
+  70: string;
+  80: string;
+  90: string;
```

Comment:
I know the figma palette differs from materials but we should try to rectify that. e.g. material already has an interface for Color that we should try to use here.

Reply:
I guess I'm a bit confused on a plan of action here. Are you saying we should be using the material colors like this example? 
`import { purple, red } from '@mui/material/colors';

const primary = red[500]; // #f44336
const accent = purple['A200']; // #e040fb
const accent = purple.A200; // #e040fb (alternative method)
`
Or are you saying we should be putting these values into a material theme / palette, like this?

`
import { createTheme } from '@mui/material/styles';
import blue from '@mui/material/colors/blue';

const theme = createTheme({
  palette: {
    primary: blue,
  },
});
`
I found this site https://bareynol.github.io/mui-theme-creator/ which I guess could help with this. 

The idea behind this PR was to have a central file so that in other places in Clutch instead of hardcoding the colors like `x = "#F12345"` we could just have `x =  pallete.Neutral.10 . A step after this would be actually putting the colors into a palette like above.

Reply:
yeah, the idea of using a palette throughout makes sense but I think you'll run into issues since material's palette expects colors to have a certain interface in order to access the various shades. e.g. 
```tsx
export interface Color {
  50: string;
  100: string;
  200: string;
  300: string;
  400: string;
  500: string;
  600: string;
  700: string;
  800: string;
  900: string;
  A100: string;
  A200: string;
  A400: string;
  A700: string;
}
```